diff --git a/CMakeLists.txt b/CMakeLists.txt
index bc162a1..fab883a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -98,7 +98,7 @@ link_directories(${LLVM_LIBRARY_DIR})
 set(LIBS ${LIBS} ${LLVM_LIBRARIES})
 add_definitions(${LLVM_CPPFLAGS})
 
-set(LLVM_SUPPORTED_VERSION "3.1")
+set(LLVM_SUPPORTED_VERSION "3.0")
 string(COMPARE NOTEQUAL "${LLVM_VERSION}" "${LLVM_SUPPORTED_VERSION}" LLVM_UNSUPPORTED_VER)
 if(${LLVM_UNSUPPORTED_VER})
 	message(FATAL_ERROR "LLVM version ${LLVM_SUPPORTED_VERSION} is required (You have ${LLVM_VERSION})")
